app/templates/components/user-sidebar.hbs

```diff
@@ -7,6 +7,11 @@
   {{/if}}
   <div class="username">{{user.username}}</div>
 </h2>
+{{#if user.biography}}
+<p class="biography">
+  {{user.biography}}
+</p>
+{{/if}}
```

Comment:
Requested change was here. Dunno how the comment failed (probably my mistake). I was saying we usually indent something like this.